Erreur lors de la compilation d'un .toml perso

`wkalamine build` plante avec cette erreur : 
```
wkalamine.exe build .\beop-1dk.toml
Creating driver DLLs █████████████████████████∙∙∙∙∙∙∙ 78%
Error while creating DLL for arch i386:

KbdTool v3.40 - convert keyboard text file to C file or a keyboard layout DLL


Compiling layout information from 'C:\Users\Olivier\Documents\TestKalamine\beop1dk.klc' for i386/x86.

    beop1dk.H : can't open for write.
   beop1dk.RC : can't open for write.
    beop1dk.C : can't open for write.
  beop1dk.DEF : can't open for write.

beop1dk.c
beop1dk.c(458) : error C2220: warning treated as error - no 'object' file generated
beop1dk.c(458) : warning C4010: single-line comment contains line-continuation character

CL.EXE returned 2

LINK : fatal error LNK1181: cannot open input file 'beop1dk.obj'

LINK.EXE returned 1181


                 4
```
[beop-1dk.txt](https://github.com/OneDeadKey/kalamine/files/14546146/beop-1dk.txt)

--- a/kalamine/template.py
+++ b/kalamine/template.py
@@ -541,7 +541,7 @@ def c_deadkeys(layout: "KeyboardLayout") -> List[str]:
             ext = hex_ord(alt)
 
             output.append(
-                f"DEADTRANS(0x{hex_ord(base)}\t, 0x{hex_ord(dk[' '])}\t, 0x{ext}\t, {dead_alt}), // {base} -> {alt}"
+                f"DEADTRANS(0x{hex_ord(base)}\t, 0x{hex_ord(dk[' '])}\t, 0x{ext}\t, {dead_alt}), /* {base} -> {alt} */"
             )
 
         output.append("")

